packages/database/migration/20260821000000_add_response_survey_finished_index/migration.sql

```
@@ -0,0 +1,11 @@
+-- The survey list aggregates response counts with `GROUP BY "surveyId", "finished"`. Without
+-- "finished" in an index that grouping cannot be served from the index alone, so Postgres falls back
+-- to a bitmap heap scan (measured ~40x the buffer traffic of the previous surveyId-only grouping).
+-- This index restores an index-only scan.
+--
+-- Keep standard CREATE INDEX here: this repo applies schema migrations through Prisma migrate deploy,
+-- which runs inside a transaction block and cannot execute CREATE INDEX CONCURRENTLY. Response writes
+-- block for the duration of the build (SHARE lock) — the same trade as every previous Response index,
+-- most recently 20250520163831_add_created_at_index_to_responses. IF NOT EXISTS keeps the migration
+-- idempotent when the index already exists (e.g. created via `prisma db push`).
+CREATE INDEX IF NOT EXISTS "Response_surveyId_finished_idx" ON "Response"("surveyId", "finished");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)